core/clause_7b_jsonschema.adoc

```diff
@@ -0,0 +1,60 @@
+[[ogc_jsonschema_level0]]
+== Requirements Class "JSON Schema Level 0"
+
+include::requirements/requirements_class_jsonschema_level0.adoc[]
+                      
+
+This standard defines four subsets of https://json-schema.org/specification.html[JSON Schema] each of increasing capability.  These subsets give server implementors flexiblity in the level of process description capability they wish to support in their server.  Servers implementing the lowest level will only allow processes with simple-valued input and output parameters to be described.  Servers that support the higest level allow processes with complex-valued input and output parameters to be described.  Servers implementing this standard can specify which level or levels of schema capability they support by specifying the appropriate conformance class URIs in the <<sc_conformance_classes,conformance section>>.
+
+This conformance class, JSON Schema Level 0, defines  the most restricted subset of https://json-schema.org/specification.html[JSON Schema] only allowing schemas that support simple types and arrays of simple types.  Specifically, the following types are supported:
+
+- boolean
+- integer
+- number
+- string
+
+and the following schema facets are supported:
+
+- minimum
+- maximum
+- exclusiveMinimum
+- exclusiveMaximum
+- enum
+- format
+- nullable
+
+include::requirements/ogc-process-description/jsonschema/REQ_jsonschema_level0.adoc[]
+
+[[ogc_jsonschema_level1]]
+== Requirements Class "JSON Schema Level 1"
+
+include::requirements/requirements_class_jsonschema_level1.adoc[]
+
+JSON Schema Level 1 supports all the capabilities of <<ogc_jsonschema_level0,level 0>> and adds support for objects.
+
+include::requirements/ogc-process-description/jsonschema/REQ_jsonschema_level1.adoc[]
+
+[[ogc_jsonschema_level2]]
+== Requirements Class "JSON Schema Level 2"
+
+include::requirements/requirements_class_jsonschema_level2.adoc[]
+
+JSON Schema Level 2 supports all the capabilities of <<ogc_jsonschema_level2,level 0>> and <<ogc_jsonschema_level1,level 1>> and adds support of an expanded set of schema facets including:
```

Comment:
``<<ogc_jsonschema_level2,level 0>>` should be `<<ogc_jsonschema_level0,level 0>>` ?